e2e/container/run_test.go

```diff
@@ -42,10 +42,12 @@ func TestRunAttachedFromRemoteImageAndRemove(t *testing.T) {
 
 func TestRunAttach(t *testing.T) {
 	skip.If(t, environment.RemoteDaemon())
+	t.Parallel()
```

Comment:
Looks  like these changes landed in the wrong commit (probably you wanted to put them in the commit before this 🙈

Reply:
ahhhhhh 🤦‍♀️ that's what I get for trying to do 3 things at the same time.